e2e-tests/.eslintrc.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
     ],
   },
   parserOptions: {
-    ecmaVersion: 12, // es2021
+    ecmaVersion: 14,
```

Comment:
We need ecma script 13 support for the private class properties I have used in `RandomValueGenerator` (the hashes)